src/jit/CMakeLists.txt

```diff
@@ -79,6 +79,15 @@ elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
     codegenarm.cpp
     unwindarm.cpp
   )
+elseif(CLR_CMAKE_PLATFORM_ARCH_I386)
+  set( ARCH_SOURCES
+    emitxarch.cpp
+    targetx86.cpp
+    lowerxarch.cpp
+    codegenxarch.cpp
+    codegenlegacy.cpp
```

Comment:
As I understand `codegenlegacy.cpp` and `stackfp.cpp` any only needed for the legacy JIT backend, not RyuJIT. See https://github.com/dotnet/coreclr/blob/master/src/jit/jit.settings.targets#L70